airbyte-connector-builder-server/connector_builder/impl/default_api.py

```diff
@@ -199,6 +198,6 @@ def _create_response_from_log_message(self, log_message: AirbyteLogMessage) -> O
     def _create_low_code_adapter(manifest: Dict[str, Any]) -> LowCodeSourceAdapter:
         try:
             return LowCodeSourceAdapter(manifest=manifest)
-        except ValidationError as error:
+        except Exception as error:
             # TODO: We're temporarily using FastAPI's default exception model. Ideally we should use exceptions defined in the OpenAPI spec
-            raise HTTPException(status_code=400, detail=f"Invalid connector manifest with error: {error.message}")
+            raise HTTPException(status_code=400, detail=f"Invalid connector manifest with error: {error.args[0]}")
```

Comment:
are the any advantages to using `error.args[0]` instead of `str(error)`? The later seems safer since args could be an empty array

Reply:
probably not and agree, str is safer, fixed!